mmaction/datasets/activitynet_dataset.py

```diff
@@ -199,7 +199,9 @@ def evaluate(
             results (list[dict]): Output results.
             metrics (str | sequence[str]): Metrics to be performed.
                 Defaults: 'AR@AN'.
-            metric_options (dict): Dict for metric options.
+            metric_options (dict): Dict for metric options. Options are
```

Comment:
provide the real default value
`default: {'AR@AN': dict( max_avg_proposals=100, temporal_iou_thresholds=np.linspace(0.5, 0.95, 10))}`